GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3019,33 +3019,42 @@ public void ChangeTerminalActiveFolder(string path)
             {
                 if (PathUtil.TryConvertWindowsPathToPosix(path, out var posixPath))
                 {
-                    ClearTerminalCommandLineAndRunCommand("cd " + posixPath);
+                    ClearTerminalCommandLineWithMacroAndRunCommand("cd " + posixPath, "#");
                 }
             }
-            else if (AppSettings.ConEmuTerminal.ValueOrDefault == "powershell")
-            {
-                ClearTerminalCommandLineAndRunCommand("cd \"" + path + "\"");
-            }
             else
             {
-                ClearTerminalCommandLineAndRunCommand("cd /D \"" + path + "\"");
+                if (AppSettings.ConEmuTerminal.ValueOrDefault == "cmd")
```

Comment:
Why not `else if`?

In fact why not change it to a `switch` statement?
```cs
                switch (AppSettings.ConEmuTerminal.ValueOrDefault)
                {
                    case "cmd":
						ClearTerminalCommandLineWithEscapeAndRunCommand("cd /D \"" + path + "\"");
                        return;

                    case "powershell":
						ClearTerminalCommandLineWithEscapeAndRunCommand("cd \"" + path + "\"");
                        return;

                    default:
                        ClearTerminalCommandLineWithMacroAndRunCommand("cd " + posixPath, "#");
                        return;
                }
```

Reply:
Thanks, done. I made the default a no-op, in case we get some unexpected value.